### PR TITLE
Fix potential capitalization error

### PR DESCRIPTION
**Pull request by rickie**
I'm not sure whether you agree with the change, but it's worth the try 😉 .
The purpose is to be the same as the `Or` two lines up.

### CODE REVIEW

**Comment by melvio** [comment]
Haha... Saw this myself too. 
But your PR comes for my own:  https://github.com/melvio/hopla/pull/109

**Comment by melvio** [comment]
Noticed that you didn't see the `CONTRIBUTING.md` because I mistakenly placed it in `.github/docs` instead of in `.github/`. 
So I just fixed that with https://github.com/melvio/hopla/pull/110.

**Review by melvio — COMMENTED** [review]
Welcome to the hall of legendary contributors :tada:

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Clone this repository by running:
 git clone "git@github.com:melvio/hopla.git"
 # Or, by using HTTPS:
 git clone "https://github.com/melvio/hopla.git"
-# or (my favorite), by using gh:
+# Or (my favorite), by using gh:
```
[the remark this answers is not in the document]
Comment (rickie) [reply README.md:24]: 😄

**Pull request merged**